components/atom/input/src/index.scss

```diff
@@ -1,5 +1,73 @@
 @import '~@schibstedspain/sui-theme/lib/settings';
 @import '~@schibstedspain/sui-theme/lib/index';
 
-.sui-AtomInput {
+$validation: success $c-success, error $c-error;
+$size: medium 40px, small 30px;
+
+@mixin input {
+  border-color: $c-gray-light;
+  border-style: solid;
+  border-width: 1px;
+  font-size: $fz-base;
+  height: 40px;
+  padding-left: $p-l;
+  padding-right: $p-l;
+  width: 100%;
+
+  &[placeholder] {
```

Comment:
Attribute selectors very innefficient. Moreover, here we have doubles like '[disabled][placeholder]'. Better use classes to identify sub elements